projects/kit/components/select/select-option/select-option.style.less

```diff
@@ -0,0 +1,21 @@
+:host(._value)::after {
```

Comment:
nit: we usually do not use ( ) when writing :host, simply :host._value

Reply:
https://github.com/taiga-family/taiga-ui/pull/10642/commits/f83d1f83aa343cdf442eebaf72b4d169d1fdc43a